### ISSUE TITLE
data set list display glitch

### ISSUE THREAD

**Issue by bernt-matthias** [galaxyproject/galaxy#5833]
A user reported this screenshot to me.

![fgpgblaodpcekbfa](https://user-images.githubusercontent.com/25689525/38259235-d320c6c4-3764-11e8-9508-99ea4631f89a.png)

I guess that the simultaneous display of two data set lists is unintended. (running 17.09)

edit by martenson: to reproduce use Firefox browser

**Comment by mblue9** [comment]
Yep, also seeing this split-screen weirdness in 17.09 right now.

**Comment by martenson** [comment]
How do I reproduce this?

**Comment by mblue9** [comment]
@martenson not sure how you can reproduce this but just to note I'm seeing it with 18.05 in usegalaxy.eu. I can't remember all the steps I made but just noticed that when I went to rerun a job from the history and clicked Execute I had the weird split-screen again, see below:

<img width="906" alt="screen shot 2018-07-11 at 1 02 57 pm" src="https://user-images.githubusercontent.com/5591821/42548672-a5c756bc-850b-11e8-9f75-9c4a953daa66.png">

**Comment by bernt-matthias** [comment]
I guess I found a way to reproduce: 
- open a data set list (don't klick the "back to history link)
- select and start any tool 

then you have the split screen as shown in the screenshot provided by @mblue9. if you open the list again you may see the header twice. 

So far I could not reproduce the split screen of my first screen shot..

**Comment by martenson** [comment]
@bernt-matthias that did not work for me, can you reproduce this on usegalaxy.org?
![screenshot 2018-07-18 11 29 55](https://user-images.githubusercontent.com/1814954/42891674-ed3b51de-8a7d-11e8-966b-7dc130d15a86.png)

**Comment by bernt-matthias** [comment]
yep usegalaxy.org and .eu. 

![screenshot from 2018-07-18 20-22-55](https://user-images.githubusercontent.com/25689525/42900389-136b0b7e-8ac9-11e8-817b-96ee847912dc.png)
![screenshot from 2018-07-18 20-26-15](https://user-images.githubusercontent.com/25689525/42900394-143490fc-8ac9-11e8-921d-9d70cd944519.png)

steps to reproduce. 
- create 3 txt data sets (e.g. by pasting)
- make a list from two of them (I decided to hide original data sets)
- go into the list 
- start an arbitrary tool (eg. replace text) on the third text file

before clicking execute you should see the content of the list. after execute you see the "split screen" history

**Comment by bernt-matthias** [comment]
just for completeness: my instance is dev ..

**Comment by martenson** [comment]
After a bit more testing I was able to reproduce. The missing bit was using a Firefox...

**Comment by mblue9** [comment]
Just to let you know I'm seeing this glitch regularly with Chrome and .eu (and it's a little disturbing).

**Comment by bernt-matthias** [comment]
happens still on dev

**Comment by mblue9** [comment]
yeah seeing it still with 19.01
